src/pages/Community/Components/NewCommunityPage.tsx

```diff
@@ -50,15 +50,17 @@ export default function NewCommunityPage({
           </div>
           <div>
             <p
-              className="text-center mt-10 font-bold text-2xl 
+              className="text-color-0 text-center mt-10 font-bold text-2xl 
             max-w-lg mx-auto">
               This is a new community, be the first to post something!
             </p>
             <div className="flex justify-center mt-10 transform scale-125">
               <Link to={`/c/${community?.cid}?post=new-post`}>
-                <button type="button" className="btn-filled rounded-3xl py-2.5">
+                <button
+                  type="button"
+                  className="button-action rounded-3xl py-2.5 px-5">
                   <p className="flex items-center gap-1 ">
-                    <PlusIcon className="h-5" /> Post
+                    <PlusIcon className="h-5 -ml-1" /> Post
```

Comment:
See my comments on src/pages/Community/Components/PostDisplayList.tsx about negative margins.

Reply:
it is just for this icon which has some kind of margin so it looks a bit off-centered